Sources/StreamChatSwiftUI/DefaultViewFactory.swift

```diff
@@ -308,7 +308,16 @@ extension ViewFactory {
     public func makeChannelLoadingView() -> some View {
         LoadingView()
     }
-    
+
+    public func makeChannelBackground() -> some View {
+        let colors = InjectedValues[\.colors]
+        return isIphone ?
+            Color(colors.background)
+            .ignoresSafeArea(.keyboard)
+            .allowsHitTesting(false)
+            : nil
+    }
```

Comment:
Suprisingly, this sets the background color of the composer only, not the whole ChatChannelView, so we might need to change the name here